app/src/main/java/pl/arturborowiec/kursakademiaandroida/core/di/NetworkModule.kt

```diff
@@ -0,0 +1,35 @@
+package pl.arturborowiec.kursakademiaandroida.core.di
+
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import org.koin.android.BuildConfig
+import org.koin.dsl.module
+import pl.arturborowiec.kursakademiaandroida.core.feature.data.RickAndMortyApi
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+val networkModule = module {
+
+    single<Interceptor> {
+        HttpLoggingInterceptor().apply {
+            if (BuildConfig.DEBUG) HttpLoggingInterceptor.Level.BODY
```

Comment:
Tutaj wynik tego bloki if else powinien być przypisany do zmiennej `level`. Gdzieś mi to umknęło na wideo, ale dodałem już komentarz pod materiałem na ten temat.

Reply:
Dobry sposób na sprawdzenie czujności widzów :)